packages/cyberstorm/src/components/Markdown/Markdown.module.css

```diff
@@ -0,0 +1,145 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  color: var(--color-text--default);
+}
+
+/*
+  TODO: Use proper styles instead of these placeholder values. The
+  content rendered on this page should match how the content is actually
+  rendered on the package page. I don't think Figma contains all the
+  stuff supported by markdown, so this needs to be discussed further and
+  all the styles should be updated all at once.
+
+  We can't control the elements inside the Markdown wrapper, e.g. to
+  give them class names, and linting requires pure selectors, so repeat
+  the .root class name on each selector.
+*/
+.root h1,
+.root h2,
+.root h3,
+.root h4,
+.root h5,
+.root h6 {
+  display: flex;
+  gap: 0.5rem;
+  align-items: flex-start;
+  align-self: stretch;
+  padding: 2rem 0 0.5rem;
+  border-bottom: 1px solid #29295b;
+  font: var(--font-body);
+  font-weight: 700;
+  line-height: normal;
+}
+
+.root h1 { font-size: 2.25rem; }
+.root h2 { font-size: 1.75rem; }
+.root h3 { font-size: 1.25rem; }
+.root h4 { font-size: 1rem; }
+.root h5 { font-size: 0.875rem; }
+
+.root h6 {
+  color: #9c9cc4;
+  font-size: 0.875rem;
+}
+
+.root ul,
+.root ol {
+  font: var(--font-body);
+  font-weight: 500;
+  line-height: 1.6rem;
+}
+
+.root ul li,
+.root ol li {
+  margin-left: 0.75rem;
+}
+
+.root img { display: inline-block; }
+
+.root li > ul,
+.root li > ol {
+  margin-top: 0.25rem;
+}
+
+.root pre {
+  padding: 1rem;
+  border: 1px solid #333370;
+  border-radius: 0.5rem;
+  background: #0f0f1f;
+}
+
+.root code {
+  display: inline-flex;
+  padding: 0.125rem 0.5rem;
```

Comment:
The flow more fluently within text and not cause any line-height discrepancies I recommend adjusting the padding to something like `padding:0 .35rem;`